src/main/java/racingcar/outputManager/ConsoleOutputManager.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.outputManager;
+
+import java.util.List;
+import racingcar.car.Car;
+
+public class ConsoleOutputManager implements OutputManager {
+    @Override
+    public void printExecutionResult(List<Car> progressResult) {
+        System.out.println();
+
+        for (Car car : progressResult) {
+            int distance = car.getDistance();
+            String name = car.getName();
+            printResult(distance, name);
+        }
+    }
+
+    private void printResult(int distance, String name) {
+        System.out.print(name + " : ");
+
+        for (int i = 0; i < distance; i++) {
+            System.out.print("-");
+        }
+
+        System.out.println();
+    }
+
+    @Override
+    public void printWinner(List<Car> winner) {
+        System.out.println();
+        System.out.print("최종 우승자 : ");
+
+        for (int i = 0; i < winner.size(); i++) {
+            printName(winner, i);
+        }
+    }
+
+    private void printName(List<Car> winner, int index) {
+        if (index != winner.size() - 1) {
+            System.out.print(winner.get(index).getName() + ", ");
+        } else {
+            System.out.print(winner.get(index).getName());
+        }
```

Comment:
```suggestion
  private void printName(List<Car> winners) {
    String result = winners.stream()
                           .map(Car::getName)
                           .collect(Collectors.joining(", "));
    System.out.print(result);
}

```
else문은 지양하는게 좋습니다! 여기서는 stream을 사용할 수 있을 것 같습니다.